### ISSUE TITLE
Fix shaping for Safari

### ISSUE THREAD

**Issue by davelab6** [VanillaandCream/Palanquin#10]
When I use Safari Version 7.1.6 on Mac OS X 10.9 to review http://www.impallari.com/testing/index-devanagari.php#sample_one (you need to click the 'Sample 1' tab link near the top right) then the shaping isn't working :(

**Comment by VanillaandCream** [comment]
Yes, The Rakaar forms are not working on 7.1.6. Any suggestions on what I should be looking at?

**Comment by davelab6** [comment]
Its निषिद्ध

**Comment by VanillaandCream** [comment]
I dont have that ligature because of weight issues., it is not related to the browser at all. I can compile a list of missing ligatures and integrate it in version 2, which I will start working on by August. Best.

**Comment by davelab6** [comment]
Cool! Will make an interesting story :)
